ngx-nuts-and-bolts-docs/docs/enum-property.md

```diff
@@ -69,3 +69,21 @@ export class ExampleComponent {
 ```
 
 Here, since `directionsData` doesn't have `differentKey` property for any of the enum values, `undefined` would be returned.
+If the need for retrieving enum property value arises you can use `getEnumPropertyValue()` function directly in your .ts file without the need for instantiating new EnumPropertyPipe instance.
+
+```ts
+import { getEnumPropertyValue } from '@infinum/ngx-nuts-and-bolts';
+
+@Component({
+	selector: 'app-example',
+	template: ``,
+})
+export class ExampleComponent {
+	public enumValue = Directions.NORTH;
+	public directionsData = directionsData;
+
+	private demoMethod(): void {
+		const enumPropertyValue = getEnumPropertyValue(Directions.NORTH, directionsData);
```

Comment:
Your examples are not the same in these two .md files. The example from the other file seems better to me.